### PR TITLE
Flaky UI components tests fix part 2

### PR DESCRIPTION
**Pull request by vhuseinova-msft**
# What

# Why

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

**Comment by vhuseinova-msft** [comment]
Added breaking change label as after merging main in, strings files were changed.

File: packages/react-components/tests/browser/VideoGallery.spec.tsx [hunk 1]
```diff
@@ -145,6 +154,11 @@ test.describe('VGL - VideoGallery tests', () => {
...
       </Stack>
     );
+    await checkVideoGalleryVisible(component);
     await expect(component).toHaveScreenshot('VGL-4-2-videogallery-after-spotlight.png');
   });
 });
+
+const checkVideoGalleryVisible = async (component: Locator): Promise<void> => {
+  await component.locator('[data-ui-id="video-gallery"]').waitFor({ state: 'visible' });
+};
```
Comment (mgamis-msft) [inline packages/react-components/tests/browser/VideoGallery.spec.tsx:164]: Thanks for adding this helper to stabilize these tests Viktoriia!

**Review by mgamis-msft — APPROVED** [review]

File: packages/react-components/tests/browser/VideoGallery.spec.tsx [hunk 2]
```diff
@@ -6,12 +6,9 @@ import { test, expect } from '@playwright/experimental-ct-react';
 import { VideoGallery } from '../../src/components/VideoGallery';
 import { VideoGalleryLocalParticipant, VideoGalleryRemoteParticipant } from '../../src';
 import { Stack } from '@fluentui/react';
+import { Locator } from 'playwright-core';
 
 test.describe('VGL - VideoGallery tests', () => {
-  test.beforeEach(async ({ page }) => {
-    await page.evaluate(() => document.fonts.ready);
-  });
```
Comment (JoshuaLai) [inline packages/react-components/tests/browser/VideoGallery.spec.tsx:13]: Why are we removing this beforeEach?

Comment (vhuseinova-msft) [reply packages/react-components/tests/browser/VideoGallery.spec.tsx:13]: We should do this after the mount, to ensure that the fonts used by the component are fully loaded.
I've moved this to tests instead

**Comment by JoshuaLai** [comment]
Just minor comment, there are a lot of empty space changes in the localize string files. Can we not include those in this PR

**Review by jimchou-dev — APPROVED** [review]
super nit: maybe don't commit the localization files in the future if they aren't actually changed to make it easier to review

**Review by JoshuaLai — APPROVED** [review]

**Comment by vhuseinova-msft** [comment]
> Just minor comment, there are a lot of empty space changes in the localize string files. Can we not include those in this PR

This is something that was added after main was merged in....

**Review by palatter — APPROVED** [review]
+1 about not including localization files

**Pull request merged**